docs/sources/datasources/jaeger/_index.md

```diff
@@ -188,10 +211,21 @@ You can query by trace ID, or use the search form to find traces.
 | **Service**      | Returns a list of services.                                                                                                       |
 | **Operation**    | Populated when you select a service with related operations. Select `All` to query all operations.                                |
 | **Tags**         | Sets tags with values in the [logfmt](https://brandur.org/logfmt) format, such as `error=true db.statement="select * from User"`. |
-| **Min Duration** | Filters all traces with a duration higher than the set value. Valid values are `1.2s, 100ms, 500us`.                              |
-| **Max Duration** | Filters all traces with a duration lower than the set value. Valid values are `1.2s, 100ms, 500us`.                               |
+| **Min Duration** | Filters all traces with a duration higher than the set value. Possible values are `1.2s, 100ms, 500us`.                           |
+| **Max Duration** | Filters all traces with a duration lower than the set value. Possible values are `1.2s, 100ms, 500us`.                            |
 | **Limit**        | Limits the number of traces returned.                                                                                             |
 
+{{< figure src="/static/img/docs/explore/jaeger-search-form.png" class="docs-image--no-shadow" caption="Screenshot of the Jaeger query editor" >}}
+
+### Query by trace ID
+
+**To query a particular trace:**
```

Comment:
Bolding the sentence treats it as a heading, but it's not a heading, so I removed the bold here and in a few other places. 
```suggestion
To query a particular trace:
```